uninstaller/create-uninstaller.ps1

```diff
@@ -2,11 +2,12 @@
 $timeStamp = $args[1]
 $versionString = $args[2]
 
+$powershell = which powershell
```

Comment:
This does not work for me on Windows 10.  It should be:

``` powershell
$powershell = where.exe powershell
```

It should be `where.exe` not 'where', because `where` is an alias for powershell's `where-object` cmdlet.
Please verify if it works for Windows 7.

Reply:
it's working on Windows 7.